controllers/novels.js

```diff
@@ -0,0 +1,46 @@
+const models = require('../models')
+
+const getAllNovels = async (request, response) => {
+  try {
+    const novels = await models.Novels.findAll({
+      include: [
+        { model: models.Authors },
+        { model: models.Genres }
+      ]
+    })
+
+    return response.send(novels)
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve novel list, please try again.')
+  }
+}
+
+const getNovelsByIdentifier = async (request, response) => {
```

Comment:
This route throws an error

![image](https://user-images.githubusercontent.com/5775716/82481449-4cf9af80-9aa3-11ea-9648-f832db2a803b.png)